core/backend/src/assets/Settings/Base.Schema.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Base.Schema.json ought to be in Settings/Schemas, no? Currently it gets loaded as a settings dictionary instead of as a schema.

Reply:
Moved it.

Reply:
Yikes! Maybe we should make a different extension for schema files? The downside is that VSCode, etc. won't treat it as JSON.